src/pages/GetInvolved/Club.js

```diff
@@ -14,6 +14,8 @@ import React from "react";
 import { useSelector } from "react-redux";
 import { withRouter, Link } from "react-router-dom";
 
+import Breadcrumbs from "../../components/Breadcrumbs/Breadcrumbs";
```

Comment:
No need for relative path

```suggestion
import Breadcrumbs from "components/Breadcrumbs/Breadcrumbs";
```